ltr/lambda_rank.py

```diff
@@ -8,25 +8,25 @@
 import paddle.v2 as paddle
 
 
-def lambda_rank(input_dim):
+def lambda_rank(input_dim, is_infer):
     """
-    lambda_rank is a Listwise rank model, the input data and label
+    Lambda_rank is a Listwise rank model, the input data and label
     must be sequences.
 
     https://papers.nips.cc/paper/2971-learning-to-rank-with-nonsmooth-cost-functions.pdf
     parameters :
       input_dim, one document's dense feature vector dimension
 
-    format of the dense_vector_sequence:
+    Format of the dense_vector_sequence:
```

Comment:
The format of the dense_vector_sequence is as follows:

Reply:
Done